e2e/src/definition.spec.ts

```diff
@@ -0,0 +1,77 @@
+import assert = require('assert');
+import { Position, Range } from 'vscode';
+import { activateAndWait, getDocUri, triggerDefinition } from './helper';
+
+const refSqlDocUri = getDocUri('ref_sql.sql');
+const packageRefDocUri = getDocUri('package_ref.sql');
+
+suite('ref definitions', () => {
+  test('Should suggest definitions for ref without package', async () => {
+    // arrange
+    await activateAndWait(refSqlDocUri);
+
+    // act
+    const definitions = await triggerDefinition(refSqlDocUri, new Position(1, 24));
+
+    // assert
+    assert.strictEqual(definitions.length, 1);
+    assert.ok(definitions[0].targetUri.path.endsWith('/test-fixture/models/table_exists.sql'));
+    assert.deepStrictEqual(definitions[0].originSelectionRange, new Range(new Position(1, 19), new Position(1, 31)));
```

Comment:
`assertThat(definitions[0].originSelectionRange, new Range(1, 19, 1, 31));`

The shorter version of creating range: `new Range(1, 19, 1, 31)`